<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>861</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border: 0px solid #8f8f91;
    border-radius: 0px;
    background-color:#303033 ;
    min-width: 80px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="com_select">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serial_connect_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="graph_s1">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
     </property>
     <property name="text">
      <string>Graph</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serial_input_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: #ffff;
    min-width: 80px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="s1_output">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 18pt &quot;MS Reference Sans Serif&quot;;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
    border: 2px solid #8f8f91;
    border-radius: 1px;
    background-color:#ffff ;
    min-width: 80px;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="serial_connect_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_select_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serial_input_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: #ffff;
    min-width: 80px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="graph_s1_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
     </property>
     <property name="text">
      <string>Graph</string>
     </property>
    </widget>
    <widget class="QLabel" name="s1_output_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 18pt &quot;MS Reference Sans Serif&quot;;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
    border: 2px solid #8f8f91;
    border-radius: 1px;
    background-color:#ffff ;
    min-width: 80px;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_3</zorder>
    <zorder>com_select</zorder>
    <zorder>serial_connect_1</zorder>
    <zorder>graph_s1</zorder>
    <zorder>serial_input_1</zorder>
    <zorder>frame</zorder>
    <zorder>s1_output</zorder>
    <zorder>graphWidget</zorder>
    <zorder>serial_connect_2</zorder>
    <zorder>com_select_2</zorder>
    <zorder>serial_input_2</zorder>
    <zorder>graph_s1_2</zorder>
    <zorder>s1_output_2</zorder>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
